### PR TITLE
Add mainnet upgrade docs

### PR DESCRIPTION
**Pull request by akhilkumarpilli**

### CODE REVIEW

File: guides/upgrade-mainnet.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+- github.com/ChainSafe/go-schnorrkel@v0.0.0-20200405005733-88cbf1b4c40d
+- github.com/Workiva/go-datastructures@v1.0.52
+....
+```
+
+**Note**: `akashd` and `akashctl` are merged into a single binary, `akash` now.
+
+8.  Migrate exported state from the current v0.8.3 version to the new v0.10.1 version:
+```
+$ akash migrate v0.40 old_genesis_updated.json --chain-id=akashnet-2 > 040_migrated_genesis.json
```
Comment (akhilkumarpilli) [inline guides/upgrade-mainnet.md:84]: @anilCSE Can I add `genesis-time` flag here in this command?

**Review by boz — APPROVED** [review]
:fire: :100: :rocket:

File: guides/upgrade-mainnet.md (hunk 1 above)
Comment (anilCSE) [reply guides/upgrade-mainnet.md:84]: May be for unique hash, let's add genesis time as March 8th 1500 UTC

Comment (anilCSE) [reply guides/upgrade-mainnet.md:84]: Actually we don't need it. We gonna export the state at some height. So block commit time should be same

**Review by troian — APPROVED** [review]

**Review by hydrogen18 — APPROVED** [review]

**Pull request merged**